File: src/main/java/ShuffleOne.java
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Paths;
import java.util.ArrayList;
import java.util.Collections;
import java.util.List;
import java.util.Random;

public class ShuffleOne {
  public static void main(String[] args) throws IOException {
    List<String> lemario = Files.readAllLines(Paths.get("src/main/resources/data/lemario.txt"));
    List<String> output = new ArrayList<>(lemario.size());
    output.addAll(lemario);

    Collections.shuffle(output, new Random(42L));
    output.subList(0, 10).forEach(System.out::println);
  }
}

File: src/main/js/shuffle_one.js
const shuffle = require('fisher-yates');
const ParkMiller = require('park-miller');
const fs = require('fs');

const lemario = fs.readFileSync('../resources/data/lemario.txt', 'utf8').split("\n");
const random = new ParkMiller(9965168727);

// Got to use old function because shuffle
// depends on scope switching
function nextFloat() {
  return random.float();
}

shuffle(lemario, nextFloat)
    .slice(0, 10)
    .forEach(line => console.log(line));
